src/ducks/SANCharts/metrics/submetrics.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { Line } from 'recharts'
+
+export const SOCIAL_TWITTER_INTERVALS = ['24h', '7d']
+export const TWITTER_FOLLOWERS_TYPE = 'twitter_followers'
+
+export const Submetrics = {
+  [TWITTER_FOLLOWERS_TYPE]: SOCIAL_TWITTER_INTERVALS.map(interval => {
```

Comment:
`TWITTER_FOLLOWERS_TYPE` indirectly references twitter metric. `Metrics.twitter.key` should be used instead